src/App.js

```diff
@@ -231,6 +234,14 @@ class App extends Component {
   handleNavigatorRef = navigatorRef =>
     Navigation.setTopLevelNavigator(navigatorRef);
 
+  handleTransactionConfirmed = () => {
```

Comment:
This is now triggered not just for speed up and cancels, but all txns from the user. Could have a situation where the zerion data (if it's delayed) clobbers the "watchPendingTransaction" results from Infura.

I'm starting to think that the append/change/removed technically work - it's just that we can't tell bc zerion is so behind.